scripts/gen_release_notes/tests/test_helper.py

```diff
@@ -0,0 +1,152 @@
+"""
+Test function to check if the methods declared in helper work as intended
+
+"""
+from unittest.mock import patch
+
+import pytest
+from gen_release_notes import generate
+from gen_release_notes.helper import helper
+
+sample_type_changes = ["env", "packer", "somemysteryvalue"]
+
+sample_string_parsers = [
+    {
+        "key": "RC/0.1",
+        "expected_value": "stg",
+    },
+    {"key": "https://github.com/some/repo", "expected_value": "url"},
+    {"key": "0.1.0", "expected_value": "prod"},
+    {"key": "someunknownvalue", "expected_value": None},
+]
+
+sample_regex_match = [
+    {"regex": r"^helloworld$", "context": "helloworld", "expected_result": True},
```

Comment:
Cool thanks for including the `expected_result` bool to ensure the test assertion is specific 🙂